Cargo.toml

```diff
@@ -19,3 +19,5 @@ opt-level = 3
 # crate, add that here. Use the form:
 #
 #     <crate> = { path = "/path/to/local/checkout" }
+html5ever = { path = "../html5ever/html5ever" }
+xml5ever = { path = "../html5ever/xml5ever" }
```

Comment:
These will need to be removed.

Reply:
Is it okay to wait for servo/html5ever#312 to merge first?